addOns/spider/src/main/java/org/zaproxy/addon/spider/Spider.java

```diff
@@ -132,6 +132,9 @@ public class Spider {
      */
     private static final Pattern GIT_URL_PATTERN = Pattern.compile("\\.git/"); // case sensitive
 
+    private static final Pattern DS_STORE_URL_PATTERN =
+            Pattern.compile("\\.DS_Store/"); // case sensitive
```

Comment:
This comment and following are unnecessary.

Reply:
Did you mean the previous occurrences of the same comment?

Reply:
I was referring to `And add '.DS_Store' as a seed, for .DS_Store based spidering`.